client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -117,6 +148,23 @@ export default {
             };
             axios.post(url, data).catch(this.handleError);
         },
+        clickedDatatypeChange: function (selectedDatatype) {
+            const url = prependPath(`/api/histories/${this.historyId}/contents/bulk`);
```

Comment:
Should be in a separate file with an async export function like other exports, which simplifies testing and modularity.